compiler/z/objectfmt/OMRELFJitCodeObjectFormat.cpp

```diff
@@ -51,14 +54,16 @@ OMR::Z::ELFJitCodeObjectFormat::emitGlobalFunctionCall(TR::GlobalFunctionCallDat
    TR::Node *callNode = data.callNode;
    TR_ASSERT_FATAL(targetAddress != NULL, "Unable to make a call for n%dn %s as targetAddress for the Call is not found.", callNode->getGlobalIndex(), callNode->getOpCode().getName());
 
-   ccGlobalFunctionData *ccGlobalFunctionDataAddress =
-      reinterpret_cast<ccGlobalFunctionData *>(cg->allocateCodeMemory(sizeof(ccGlobalFunctionData), false));
-
-   if (!ccGlobalFunctionDataAddress)
+   OMR::CCData *codeCacheData = cg->getCodeCache()->manager()->getCodeCacheData();
+   OMR::CCData::index_t index;
+   
+   if (!(codeCacheData->put(NULL, sizeof(ccGlobalFunctionData), alignof(uintptr_t), NULL, index)))
```

Comment:
```suggestion
   if (!(codeCacheData->put(NULL, sizeof(ccGlobalFunctionData), alignof(ccGlobalFunctionData), NULL, index)))
```

Reply:
Thanks a lot for the suggestions, I think using the alignment of ccGlobalFunctionData works . Made changes in https://github.com/eclipse/openj9-omr/compare/40c06eea215a7f142827601c278ee37e63eb5e9a..bb0f01ada9bdfde593e7c36674e28c9ff28b7368